integrations.go

```diff
@@ -274,3 +274,68 @@ func (client *Client) UpdateIntegrationGCP(cir *IntegrationGCPUpdateRequest) err
 func (client *Client) DeleteIntegrationGCP(cir *IntegrationGCPDeleteRequest) error {
 	return client.doJsonRequest("DELETE", "/v1/integration/gcp", cir, nil)
 }
+
+/*
+	Azure Integration
+*/
+
+// IntegrationAzure defines the response for listing Datadog-Azure CloudPlatform integration.
+type IntegrationAzure struct {
+	TenantName  *string `json:"tenant_name"`
+	ClientID    *string `json:"client_id"`
+	HostFilters *string `json:"host_filters,omitempty"`
+}
+
+// IntegrationAzureCreateRequest defines the request payload for creating Datadog-Azure CloudPlatform integration.
+type IntegrationAzureCreateRequest struct {
+	TenantName   *string `json:"tenant_name"`
+	ClientID     *string `json:"client_id"`
+	ClientSecret *string `json:"client_secret"`
+	HostFilters  *string `json:"host_filters,omitempty"`
+}
+
+// IntegrationAzureUpdateHostFiltersRequest defines the request payload for creating Datadog-Azure CloudPlatform integration.
+type IntegrationAzureUpdateHostFiltersRequest struct {
```

Comment:
The API documentation says that `client_secret` is required for this request as well. I'd prefer making all the structures from this PR one - `IntegrationAzure` - perhaps with the exception of `IntegrationAzureUpdateAccountRequest` which has some unique arguments (`new_client_id` and `new_client_name` - BTW I couldn't find any documentation of these in the API docs, where are they defined?).

Reply:
Yes, so this is what we discussed with @Ricky-Thomas, no documentation for this endpoint just yet. 
I can however work on the `struct`s, thanks for the review.

Reply:
There are docs for that endpoint available now here: https://docs.datadoghq.com/api/?lang=bash#integration-azure